src/model/image_info.js

```diff
@@ -298,8 +305,13 @@ export default class ImageInfo {
         this.can_annotate = response.perms.canAnnotate;
         if (typeof response.meta.imageAuthor === 'string')
             this.author = response.meta.imageAuthor;
-        if (typeof response.meta.imageName === 'string')
+        if (typeof response.meta.imageName === 'string') {
```

Comment:
I think we should use the short name in the info too and use the full name in the tooltip
This can be done in another PR